gamefile.py

```diff
@@ -0,0 +1,36 @@
+import random
+import time
+
+print('Welcome to the HangMan Games!')
```

Comment:
Please do not execute code directly in the module. You should use things defined here in `game.py` instead of importing everything.